[2025-04-23T20:18:15.166+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-23T20:18:15.221+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: final_project_BBDL.LandingToBronze manual__2025-04-23T20:18:12.150090+00:00 [queued]>
[2025-04-23T20:18:15.229+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: final_project_BBDL.LandingToBronze manual__2025-04-23T20:18:12.150090+00:00 [queued]>
[2025-04-23T20:18:15.230+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-04-23T20:18:15.315+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): LandingToBronze> on 2025-04-23 20:18:12.150090+00:00
[2025-04-23T20:18:15.318+0000] {standard_task_runner.py:63} INFO - Started process 175 to run task
[2025-04-23T20:18:15.321+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'final_project_BBDL', 'LandingToBronze', 'manual__2025-04-23T20:18:12.150090+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/project_solution.py', '--cfg-path', '/tmp/tmp1u35ougo']
[2025-04-23T20:18:15.324+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask LandingToBronze
[2025-04-23T20:18:15.638+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.10/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-23T20:18:15.723+0000] {task_command.py:426} INFO - Running <TaskInstance: final_project_BBDL.LandingToBronze manual__2025-04-23T20:18:12.150090+00:00 [running]> on host 93c11fc571ef
[2025-04-23T20:18:25.516+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='final_project_BBDL' AIRFLOW_CTX_TASK_ID='LandingToBronze' AIRFLOW_CTX_EXECUTION_DATE='2025-04-23T20:18:12.150090+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-23T20:18:12.150090+00:00'
[2025-04-23T20:18:25.517+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-23T20:18:26.104+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-23T20:18:26.104+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/landing_to_bronze.py", line 32, in main
    .getOrCreate()
  File "/opt/spark/python/pyspark/sql/session.py", line 497, in getOrCreate
    sc = SparkContext.getOrCreate(sparkConf)
  File "/opt/spark/python/pyspark/context.py", line 515, in getOrCreate
    SparkContext(conf=conf or SparkConf())
  File "/opt/spark/python/pyspark/context.py", line 201, in __init__
    SparkContext._ensure_initialized(self, gateway=gateway, conf=conf)
  File "/opt/spark/python/pyspark/context.py", line 436, in _ensure_initialized
    SparkContext._gateway = gateway or launch_gateway(conf)
  File "/opt/spark/python/pyspark/java_gateway.py", line 107, in launch_gateway
    raise PySparkRuntimeError(
pyspark.errors.exceptions.base.PySparkRuntimeError: [JAVA_GATEWAY_EXITED] Java gateway process exited before sending its port number.
[2025-04-23T20:18:26.140+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=final_project_BBDL, task_id=LandingToBronze, run_id=manual__2025-04-23T20:18:12.150090+00:00, execution_date=20250423T201812, start_date=20250423T201815, end_date=20250423T201826
[2025-04-23T20:18:26.365+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 6 for task LandingToBronze ([JAVA_GATEWAY_EXITED] Java gateway process exited before sending its port number.; 175)
[2025-04-23T20:18:26.410+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-04-23T20:18:26.457+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-23T20:18:26.614+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
